Userland/Utilities/paste.cpp

```diff
@@ -76,9 +72,9 @@ int main(int argc, char* argv[])
             // Technically there's a race here...
             auto data_and_type = clipboard.fetch_data_and_type();
             if (data_and_type.mime_type.is_null()) {
-                spawn_command(watch_command, {}, "clear");
+                (void)spawn_command(watch_command, {}, "clear");
```

Comment:
Why go to all that trouble if you're gonna `(void)` it anyway?

Reply:
Not very useful, but at least `spawn_command` is cleaner. Maybe printing something to the terminal can improve things?

Reply:
Why not TRY it?

Reply:
There's no way to propagate the error inside the clipboard `onchange` callback.